index.js

```diff
@@ -77,7 +77,10 @@ module.exports = {
 
   testemMiddleware: function(app) {
     if (!this._isCoverageEnabled()) { return; }
-    attachMiddleware(app, { root: this.project.root, ui: this.ui });
+    attachMiddleware(app, {
+      root: this.project.root,
+      configDirName: path.dirname(this.project.configPath())
```

Comment:
Why are we no longer passing in `ui`?

Can you put the config path invocation above here as a variable declaration (then just pass it through here)?